Source/ALSV4_CPP/Public/Character/ALSBaseCharacter.h

```diff
@@ -625,4 +627,7 @@ class ALSV4_CPP_API AALSBaseCharacter : public ACharacter
 
 	/** We won't use curve based movement and a few other features on networked games */
 	bool bEnableNetworkOptimizations = false;
+
+private:
+	UALSDebugComponent* m_debugComponent = nullptr;
```

Comment:
Please use camel case for variable names. `DebugComponent` should be fine.

Reply:
Sorry I didn't know that the project is using the Unreal Engine coding style guide. Changed it.